app/assets/javascripts/app/components/discussion_card/templates/discussion_card.html.slim

```diff
@@ -1,7 +1,8 @@
-md-card class="Discussion Discussion--{{ discussion.status }}" ng-click="ctrl.show(discussion.id)"
-  .Discussion-title
-    | {{ discussion.title }}
-  .Discussion-subtitle
-    | {{ discussion.subtitle }}
-  .Discussion-commentsCount
+a.u-noDecoration ui-sref="discussions.show({id: discussion.id})"
```

Comment:
Then why does it have a class called `u-noDecoraition` instead of a `Dicussion-something`? Ok make it the root of discussion? I already said that you use utilities to much, you're not supposed.